build.gradle

```diff
@@ -32,9 +32,9 @@ subprojects {
         postgresqlDriverVersion = '42.3.3'
         oracleDriverVersion = '19.8.0.0'
         sqlserverDriverVersion = '8.4.1.jre8'
-        grpcVersion = '1.38.0'
-        protocVersion = '3.17.2'
-        protobufVersion = '3.17.2'
+        grpcVersion = '1.46.0'
+        protocVersion = '3.20.1'
+        protobufVersion = '3.20.1'
```

Comment:
These upgradings are backward compatible?

Reply:
Yes, I think so. To confirm it, I ran the Scalar DB server integration test with the following configuration :
 - `server:integrationTestScalarDbServer`(`master` branch version) targeting Scalar DB Server (`fix_CVE-2022-25647` branch version) running on Mysql
  - `server:integrationTestScalarDbServer`(`fix_CVE-2022-25647` branch version) targeting Scalar DB Server (`master` branch version) running on Mysql

Reply:
Thank you for the confirmation!